app/bolt_listeners.py

```diff
@@ -138,22 +157,37 @@ def respond_to_app_mention(
                 user=context.user_id,
             )
         else:
-            steam = start_receiving_openai_response(
-                openai_api_key=openai_api_key,
-                model=context["OPENAI_MODEL"],
-                messages=messages,
-                user=context.user_id,
-            )
-            consume_openai_stream_to_write_reply(
-                client=client,
-                wip_reply=wip_reply,
-                context=context,
-                user_id=user_id,
-                messages=messages,
-                steam=steam,
-                timeout_seconds=OPENAI_TIMEOUT_SECONDS,
-                translate_markdown=TRANSLATE_MARKDOWN,
-            )
+            # Begin Render image
```

Comment:
Sorry to say this but I am not thinking of adding Bing image related dependencies to this app.